packages/env-utils/src/envUtils.native.ts

```diff
@@ -79,7 +79,13 @@ const getOsNameWeb = () => '';
 
 const getOsFamily = (): 'Linux' => 'Linux';
 
-export const getJWSPublicKey = () => (isCodesignBuild() ? publicKey.codesign : publicKey.dev);
+export const getJWSPublicKey = (use: JWSPublicKeyUse) => {
```

Comment:
Wouldn't it be better to also have `useCodeSignKey` for consistency?

Reply:
Sure, makes sense, and it will be used later on. So here it is: https://github.com/trezor/trezor-suite/pull/20449/commits/f1dd965e8fe9adfad717fa5622229d93d435e6f8